### DISCUSSION TITLE
Install VC redistributable from a msi

### DISCUSSION THREAD

**Discussion by RoyZhang2022** [wixtoolset/issues#7723] [Questions]
Hi,

My application has a dependency on Microsoft VC runtime libraries. So I want to include the VC runtime package in my msi installer. I added the following code to my wix file. The msi installer could be built sucessfully.  But when I tested the installer, I could see the VC runtime installation could be started, but stopped at 1/3 and failed after 1 minute. I see this error from Windows event viewer. Does anyone have any idea of what is wrong? Tried on different machines and saw same problem.

--error info
```
Error 1722. There is a problem with this Windows Installer package. A program run as part of the setup did not finish as expected. Contact your support personnel or package vendor. Action InstallVCRedist, location: C:\Windows\Installer\MSIAA36.tmp, command: /install /passive /norestart 
```

--code
```
<Wix xmlns='http://schemas.microsoft.com/wix/2006/wi'>
    <Product Name=......

        ......
        <Binary Id="VCRedist" SourceFile='@VC_REDIST_NAME@' />
        <CustomAction Id="InstallVCRedist" BinaryKey="VCRedist" ExeCommand="/install /passive /norestart" Execute="deferred" Return="check" Impersonate="no" />
        <InstallExecuteSequence>
            <Custom Action="InstallVCRedist" After="InstallFiles" />
        </InstallExecuteSequence>
    </Product>
</Wix>
```

--VC redist installation stops at 1/3
![image](https://github.com/wixtoolset/issues/assets/104782736/9fbe1481-9e56-4747-9942-79ebbb8ecf45)

**Comment by robmen** [comment]
The Windows Installer does not support concurrent installs. The VC Redist install is hung waiting for your install to finish, and your install is hung waiting for the VC Redist to finish. You are deadlocked.

This is why Bundles exist.

**Reply by RoyZhang2022** [reply]
Thanks! We will use bundles.
